pontoon/teams/static/css/team.css

```diff
@@ -123,6 +123,27 @@
   color: #7BC876;
 }
 
+.info {
+  text-align: center;
+}
+
+.edit-info {
+  float: right;
+}
+
+.editing-info {
+  width: 100%;
+  margin: 20px 0;
+}
+
+.controls {
+  text-align: right;
+}
+
+.controls .cancel {
+  margin: 9px;
+}
+
```

Comment:
These selectors are not specific enough. Since dashboards are singe page apps, these rules break the UI on other tabs. Which also means we should wrap the template into something like `#info-wrapper`.